src/translations.js

```diff
@@ -293,6 +303,7 @@ const wakatimeCardLocales = {
     sk: "Žiadna kódovacia aktivita tento týždeň",
     tr: "Bu hafta herhangi bir kod yazma aktivitesi olmadı",
     pl: "Brak aktywności w tym tygodniu",
+    uz: "Bu hafta faol bo'lmadi",
```

Comment:
@MrMuminov I don't speak uzbek but is there a reason you chose `Bu hafta faol bo'lmadi` over `
Bu hafta kodlash faoliyati yo'q` that google suggests. I guess because it is shorter?

Reply:
Google has officially translated.